kong/kong.go

```diff
@@ -40,7 +40,7 @@ func NewClient(httpClient *http.Client, config *config.KongConfiguration) *Clien
 
 //	generateAPI := &API{
 //		Name:             "sakabe",
-//		UpstreamURL:      "http://koudaiii.com",
+//		UpstreamURL:      "http://wantedly.com",
```

Comment:
- `sakabe` is still remained.
- The correct URL of our service is https://www.wantedly.com